vector/src/main/java/im/vector/app/features/onboarding/OnboardingViewModel.kt

```diff
@@ -897,6 +899,21 @@ class OnboardingViewModel @AssistedInject constructor(
     fun getFallbackUrl(forSignIn: Boolean, deviceId: String?): String? {
         return authenticationService.getFallbackUrl(forSignIn, deviceId)
     }
+
+    private fun updateDisplayName(displayName: String) {
+        setState { copy(asyncDisplayName = Loading()) }
```

Comment:
this uses the same async state mechanism as the other parts of the onboarding state (sign in/register), the login V2 converts this to a simple `isLoading: boolean` but I've avoided doing the same change here to keep the diff smaller